released/packages/coq-haskell/coq-haskell.1.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "1.1.0"
```

Comment:
```suggestion
```
This is already part of the file path, so not needed in the `opam` file itself. Makes it easier to copy opam packages.